client-app/pages/company/info.vue

```diff
@@ -29,12 +27,49 @@
             </template>
           </VcInput>
         </div>
+      </VcWidget>
+      <VcWidget
+        v-if="userCanEditOrganization || !loadingOrganization || !loadingUser"
+        :title="$t('pages.company.info.labels.company_logo')"
+        size="lg"
+      >
+        <div class="flex gap-4">
+          <div v-if="logoUrl" class="flex flex-col justify-between">
+            <div>
+              <VcImage :alt="$t('pages.company.info.labels.company_logo')" :src="logoUrl" class="logo-preview__image" />
```

Comment:
1. You don't have a `logo-preview` block here, so you cannot use the `logo-preview__image` element.  
     → Use Tailwind classes directly instead.  
     Reference: https://tailwindcss.com/docs  

  2. If we follow BEM methodology, the BEM block name should match the file name.  
     → Check other components for consistency.  
     Reference: https://getbem.com/

Reply:
fixed